File: Bronze/10818/10818.cpp
// [10818] 최소, 최대
#include <iostream>
#include <algorithm>
using namespace std;

int main(void) {
    int n;
    cin >> n;
    int *arr = new int[n];
	  
    for (int i = 0; i < n; i++) {
        cin >> arr[i];
    }
    
	// 정렬 후 출력
    sort(arr, arr+n);
    cout << arr[0] << " " << arr[n-1];
}

File: Bronze/10818/10818.java
// [10818] 최소, 최대
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.Arrays;

public class Main {
    public static void main(String[] args) throws IOException
    {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int N = Integer.parseInt(br.readLine());
        String[] s = br.readLine().split(" ");

        // int형 배열로 옮기기
        int[] arr = new int[N];
        for (int i = 0; i < N; i++) {
            arr[i] = Integer.parseInt(s[i]);
        }
        Arrays.sort(arr);   // 정렬

        System.out.println(arr[0] + " " + arr[N-1]);
    }
}